src/lib/dom-api.html

```diff
@@ -593,11 +593,57 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
+        configurable: true,
+        get: function() {
+          return wrap(this.node).activeElement;
```

Comment:
Do you mean `Polymer.DomApi.wrap`? The `DomApi` constructor defined in this file doesn't have a `wrap` property - the object returned / assigned to `Polymer.DomApi` does though. However, the other function calling `wrap` calls it directly since it's defined in this scope. Thoughts?